worker/caasoperator/commands/container-spec-set.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package commands
+
+import (
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"gopkg.in/juju/names.v2"
+	"gopkg.in/yaml.v2"
+)
+
+// ContainerspecSetCommand implements the status-set command.
+type ContainerspecSetCommand struct {
+	cmd.CommandBase
+	ctx      Context
+	specYaml string
+	unitName string
+}
+
+// NewContainerspecSetCommand makes a container-spec-set command.
+func NewContainerspecSetCommand(ctx Context) (cmd.Command, error) {
+	return &ContainerspecSetCommand{ctx: ctx}, nil
+}
+
+func (c *ContainerspecSetCommand) Info() *cmd.Info {
+	doc := `
+Sets configuration data to use for a container.
+By default, the spec applies to all units for the
+application. However, if a unit name is specified,
+the spec is used for just that unit.
+`
+	return &cmd.Info{
+		Name:    "container-spec-set",
+		Args:    "<spec yaml> [<unit-name>]",
```

Comment:
make unit name a flag, since it's optional? similar to how we have a flag for specifying a relation for other commands

Reply:
ok